### ISSUE TITLE
Pollution by Python3 prevents EasyBuild from working smoothly

### ISSUE THREAD

**Issue by citibeth** [hpcugent/easybuild-framework#1532]
The problem comes when the system default "python" is python3, not python2.  This causes problems at many stages:

1. The call to 
        python bootstrap_eb.py
    doesn't work because the script calls a hardcoded "python" inside.  This can be fixed, see: http://stackoverflow.com/questions/2589711/find-full-path-of-python-interpreter

2. Once things get installed, "module EasyBuild" doesn't work.   Again, it gets confused by the system default python, rather than the one we originally launched with.

3. The eb command doesn't work either.  Same reason . I jury-rigged the following script, which is obviously not portable:
   export PYTHONUSERBASE=$EB_ROOT/software/EasyBuild/2.5.0
    $EB_ROOT/bin/python -m easybuild.main --robot=/Users/rpfische/eb/easyconfigs "$@

SUGGESTED TESTING APPROACH:

1. Install Python3 on your system and configure paths so "python" defaults to "python3."
   (Alternately: just hack your system so the "python" command does something non-funtional).
2. Follow the docs to install EasyBuild, starting with "python2.7 bootstrap_eb.py"
 3. Fix any fireworks that follow

**Comment by boegel** [comment]
@citibeth: a couple of problems related to having Python 3 as the default Python were handled in #1411, but apparently we didn't get the full picture here.

I'll try and look into playing around with it on a system where Python 3 is the default (e.g. Arch Linux), and see what problems are left unhandled.

This is very much related to #1410 btw.

**Comment by citibeth** [comment]
Even without finding an Arch Linux, you can probably catch most of the
problems with something like the following BASH environment:

```
mkdir ~/python3_prototype
echo 'exit -1' >~/python3_prootype/python
chmod a+x ~/python3_prootype/python
export PATH=~/python3_prototype:$PATH
```

**Comment by boegel** [comment]
@citibeth This should be fixed with EasyBuild v2.7.0, please verify?

**Comment by boegel** [comment]
Closing this, cfr. #1410
